frontend/src/Settings/Profiles/Release/EditReleaseProfileModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import Modal from 'Components/Modal/Modal';
+import { sizes } from 'Helpers/Props';
+import { clearPendingChanges } from 'Store/Actions/baseActions';
+import EditReleaseProfileModalContent from './EditReleaseProfileModalContent';
+
+interface EditReleaseProfileModalProps {
+  id?: number;
+  isOpen: boolean;
+  onModalClose: () => void;
+  onDeleteReleaseProfilePress?: () => void;
+}
+
+function EditReleaseProfileModal({
+  isOpen,
+  onModalClose,
+  ...otherProps
+}: EditReleaseProfileModalProps) {
+  const dispatch = useDispatch();
+
+  const onModalClosePress = useCallback(() => {
```

Comment:
Avoid conflicting with `onModalClose` with `handle` instead of `on`.

```suggestion
  const handleModalClose = useCallback(() => {
```